### DISCUSSION TITLE
Used your tool to make overlay from gpx file. Worked like a charm.

### DISCUSSION THREAD

**Discussion by pialanrveze** [time4tea/gopro-dashboard-overlay#199] [Show and tell]
Thank you for your tool and all efforts to make it simple for noobs like me. I am far from coding and programming but anyway after a couple of hours I made overlay video. 

venv/bin/gopro-dashboard.py --use-gpx-only --gpx ~/Desktop/UphillD.gpx --units-speed kph --profile overlay --profile mov --overlay-size 1920x1080 UphillD-dashboard.mov

It took me about 5-7 minutes to create overlay on Intel UHD 620 Graphics for 15 minutes long video. 

Didn't read about including and excluding widgets, ahah. So I just edit layout file and translated it to Russian. Didn't understand how translate "kph" big label. Left it as it is. 

Thanks again and all the best! 

![UphillTLT 3 этап 0-17 screenshot](https://github.com/user-attachments/assets/7e874be4-1e23-42de-be28-a86c436a4ee4)

**Comment by time4tea** [comment]
Thank you so much for the kind comments! - I'm really glad you liked the software.
